src/UI/Factory.php

```diff
@@ -336,4 +336,28 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers are used to display secondary information to a component.
+	 *   composition: >
+	 *     Popovers consist of a short title and text.
+	 *   effect: >
+	 *     Popovers are typically shown by clicking or hovering over a component. The position
+	 *     of the Popover is calculated automatically be default. However, it is possible to
+	 *     specify if the popover appears at the top, right, bottom or left of the triggerer component.
+	 * context: >
+	 * rules:
+	 *   usage:
+	 *     1: Popovers MUST contain a title and a text.
+	 *     2: The text MUST be plain text (no HTML) and no longer than a paragraph.
```

Comment:
Very incomplete. Accessability is a huge topic in the case of popovers. Please checkout the rules proposed at: [https://docs.google.com/document/d/1JoInU0-tkRHDOMjMTNziV-hJb6ypDCI1zT9OGtAThZk/edit#heading=h.um0atzynczok](https://docs.google.com/document/d/1JoInU0-tkRHDOMjMTNziV-hJb6ypDCI1zT9OGtAThZk/edit#heading=h.um0atzynczok)